lib/traject/marc_extractor.rb

```diff
@@ -105,6 +105,11 @@ module Traject
   # lazily create and then re-use a MarcExtractor object with
   # particular initialization arguments.
   class MarcExtractor
+    # constant values
+    EMPTY = []
```

Comment:
Better freeze it too. `EMPTY = [].freeze`.  Cause if anyone does accidentally modify it, it'll cause a world of hurt when everyone is sharing the same `EMPTY` array (not threadsafe either).